docs/install/README.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Install
+description: Get up and running quickly by installing a Filecoin node to your system.
+---
+
+# Install Filecoin
+
+| Mine Filecoin                                                                                       | Build with Filecoin                                                                                                                 | For everyone else                                                                                                                                          |
+| --------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Are you setting up a miner to mine Filecoin? This section is for you!<br>[Find out more ↓](#miners) | Are you a developer looking to build an application on Filecoin? This section _might_ be for you.<br>[Find out more ↓](#developers) | Are you just looking to play around with Filecoin to see what all the fuss is about? This section is not for you.<br>[Find out more ↓](#for-everyone-else) |
```

Comment:
```suggestion
```

This feels like added complexity and I'm inclined to drop it for now until we've got a nice graphic wizard/selector.